kmap/mapper.go

```diff
@@ -0,0 +1,387 @@
+package kmap
+
+import (
+	"fmt"
+
+	"github.com/infracloudio/ksearch/pkg/util"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+var rPool *ResourcePool
+var groot *Resource
+var indentationCount int = 0
+var indentation string
+
+func FindThemAll(clientset *kubernetes.Clientset, namespace *string) error {
```

Comment:
This function should have a comment explaining what it is intended to do. `Lint the code`.

Reply:
Id

> Exported functions should have a comment explaining what it is intended to do. `Lint the code`.

Added a comment about the function

Reply:
Lint the code! the comment should be of type `// FindThemAll does stuff` and why are you using `testuser-ops` to comment here?